### ISSUE TITLE
Error while using OpenAI embeddings

### ISSUE THREAD

**Issue by IhorNi** [lilacai/lilac#1036]
I constantly get error while computing OpenAI embeddings within both concept playground and while attempting to compute embeddings in a dataset

Would kindly appreciate your support.

**Error message**

```
Computing embeddings for "local/fin_statements/openai" took 55.018s.
Route error: https://shelf-target-use-case-tagging.ngrok.app/api/v1/concepts/local/fin_statements/model/openai?create_if_not_exists=false
Traceback (most recent call last):
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/tenacity/__init__.py", line 382, in __call__
    result = fn(*args, **kwargs)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/embeddings/openai.py", line 93, in embed_fn
    return [np.array(embedding['embedding'], dtype=np.float32) for embedding in response['data']]
TypeError: 'CreateEmbeddingResponse' object is not subscriptable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/router_utils.py", line 24, in custom_route_handler
    return await original_route_handler(request)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/fastapi/routing.py", line 192, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/starlette/concurrency.py", line 41, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/router_concept.py", line 202, in get_concept_model
    model = DISK_CONCEPT_MODEL_DB.sync(
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/concepts/db_concept.py", line 221, in sync
    model_updated = model.sync(concept)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/concepts/concept.py", line 317, in sync
    self._compute_embeddings(concept)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/concepts/concept.py", line 365, in _compute_embeddings
    for id, (embedding, *_) in zip(missing_ids, missing_embeddings):
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/embeddings/embedding.py", line 46, in _embed_fn
    items = embedding.compute(data)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/embeddings/openai.py", line 95, in compute
    return chunked_compute_embedding(
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/lilac/embeddings/embedding.py", line 83, in chunked_compute_embedding
    batch_embeddings = embed_fn(batch_texts)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/tenacity/__init__.py", line 289, in wrapped_f
    return self(f, *args, **kw)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/tenacity/__init__.py", line 379, in __call__
    do = self.iter(retry_state=retry_state)
  File "/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/tenacity/__init__.py", line 326, in iter
    raise retry_exc from fut.exception()
tenacity.RetryError: RetryError[<Future at 0x7ff3ed8701c0 state=finished raised TypeError>]
```

**Comment by nsthorat** [comment]
Sorry about that, just sent a fix and will release a patch fix today! Thank you for filing.

**Comment by nsthorat** [comment]
Fixed in 0.2.2: https://github.com/lilacai/lilac/releases/tag/v0.2.2
